virttest/utils_net.py

```diff
@@ -3462,15 +3462,19 @@ def get_ip_address_by_interface(ifname, ip_ver="ipv4", linklocal=False):
     :param ifname: interface name
     :param ip_ver: Host IP version, ipv4 or ipv6.
     :param linklocal: Whether ip address is local or remote
-    :raise NetError: When failed to fetch IP address (ioctl raised IOError.).
+    :raise IPAddrGetError: When failed to fetch IP address.
     """
     if ip_ver == "ipv6":
         ver = netifaces.AF_INET6
         linklocal_prefix = "fe80"
     else:
         ver = netifaces.AF_INET
         linklocal_prefix = "169.254"
-    addr = netifaces.ifaddresses(ifname).get(ver)
+    try:
+        addr = netifaces.ifaddresses(ifname).get(ver)
+    except:
```

Comment:
I want to confirm which exceptions will be raised, to see if we really need to capture the whole Exception indeed.

Reply:
Add this section as `TODO`. Please help me review it again, thanks a lot.